opensrp-chw/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">localhost</domain>
+        <domain includeSubdomains="true">192.168.43.193</domain>
```

Comment:
remove this, only localhost should be allowed

Reply:
Done.